applications/API/serializers.py

```
@@ -0,0 +1,505 @@
+from asyncio.log import logger
+from functools import lru_cache
+from typing import Any, Dict, Optional
```

Ruff (0.12.2)

`typing.Dict` is deprecated, use `dict` instead

(UP035)